src/maidsafe/vault/mpid_manager/value.cc

```diff
@@ -0,0 +1,102 @@
+/*  Copyright 2015 MaidSafe.net limited
```

Comment:
not sure, but not yet in this pull request.
However the MpidManagerValue is mainly just a wrapper of ImmutableData,
allowing universal account_transfer_handler can be used in mpid_manager

I think the MpidManger service tests will cover most of the MpidManagerValue,
so not sure whether a separate unit test for value is required.